### DISCUSSION TITLE
What's the most minimal setup?

### DISCUSSION THREAD

**Discussion by 400HPMustang** [basnijholt/adaptive-lighting#439] [Q&A] [answered]
That might be the wrong question to ask. Let me start by saying HomeKit adaptive lighting is just enabled or disabled. I'm fine with whatever it does when its enabled. 

I want the same behavior in Home Assistant. 

Is there like a documented "make it behave like HomeKit" configuration/setup?

**Comment by th3w1zard1** [comment, marked as answer]
The default configuration is widely considered the most minimal 'set it and forget it' config. You just need to add your lights to the config and it's done.

I recommend ticking `take_over_control`

**Reply by 400HPMustang** [reply]
I see so there are actual defaults set up that don't require additional configuration. That part was confusing me originally. I appreciate you taking the time to reply.
